Tools/automation.ps1

```diff
@@ -0,0 +1,790 @@
+                    contentVersion = $JSONContentVersion
+                    Parameters = @{
+                        name = @{ value = $CDBAccountName }
+                        tags = @{ 
+                            value = @{
+                                defaultExperience = "Core (SQL)"
+                                CosmosAccountType = "Production"
+                            } 
+                        }
+                        consistencyPolicy = @{
+                            value = @{
+                                defaultConsistencyLevel = "ConsistentPrefix"
+                                maxIntervalInSeconds    = 5
+                                maxStalenessPrefix      = 100
+                            }
+                        }
+                        locations = @{
+                            value = @(
+                                @{
+                                    locationName     = "West US"
+                                    failoverPriority = 0
+                                    isZoneRedundant  = $false
+                                }
+                                @{
+                                    locationName     = "East US"
+                                    failoverPriority = 1
+                                    isZoneRedundant  = $false
+                                }
+                            )
+                        }
+                        ipRules =@{
+                            value = @(
+                                @{
+                                    ipAddressOrRange = "104.42.195.92"
+                                }
+                                @{
+                                    ipAddressOrRange = "40.76.54.131"
+                                }
+                                @{
+                                    ipAddressOrRange = "52.176.6.30"
+                                }
+                                @{
+                                    ipAddressOrRange = "52.169.50.45"
+                                }
+                                @{
+                                    ipAddressOrRange = "52.187.184.26"
+                                }
+                                @{
+                                    ipAddressOrRange = "0.0.0.0"
+                                }
+                            )
+                        }
+                        backupPolicy = @{
+                            value = @{
+                                type = "Periodic"
+                                periodicModeProperties = @{
+                                        backupIntervalInMinutes        = 240
+                                        backupRetentionIntervalInHours = 720
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            @{  ObjectType = "CosmosDBDatabase"
```

Comment:
Cosmos offers a free tier for people trying things out.
It allows up to 1000Ru/s.  Only one such free tier db is allowed per subscription.
We should make it easy for users trying things out to opt into the free option and provide guidance for how to ramp up once they have validated the solution will work for them.

Reply:
+1